website/snippets/_new-sl-changes.md

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
This note is great! Any reason it's not on the Architecture page? https://deploy-preview-3705--docs-getdbt-com.netlify.app/docs/use-dbt-semantic-layer/sl-architecture

Reply:
good callout, i don't know why its not there so i must've missed it. thank you!